src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { PeopleFilters } from '../PeopleFilters/PeopleFilters';
+import { Person } from '../../types';
+import { Loader } from '../Loader/Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { useSearchParams } from 'react-router-dom';
+import { getFilteredPeople } from '../../utils/getFilteredPeople';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(setPeople)
+      .catch(() => setError('Something went wrong'))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  const [searchParams] = useSearchParams();
+  const newPeople = getFilteredPeople(people, searchParams);
+  const wrongData = ! people.length && !isLoading && !error;
```

Comment:
`isWrongData` is more appropriate name, which implies that it's boolean. Usage of `wrongData` name implies that it's some kind of data.